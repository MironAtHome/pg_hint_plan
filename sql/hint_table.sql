-- Tests for the hint table
LOAD 'pg_hint_plan';

-- Attempting to use the hint table without the extension created
-- emits a WARNING.
SET pg_hint_plan.enable_hint_table TO on;
SELECT 1;
SET pg_hint_plan.enable_hint_table TO off;

CREATE EXTENSION pg_hint_plan;
SET pg_hint_plan.enable_hint_table TO on;
SELECT 1;
SET pg_hint_plan.enable_hint_table TO off;
DROP EXTENSION pg_hint_plan;
