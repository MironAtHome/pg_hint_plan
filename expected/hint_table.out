-- Tests for the hint table
LOAD 'pg_hint_plan';
-- Attempting to use the hint table without the extension created
-- emits a WARNING.
SET pg_hint_plan.enable_hint_table TO on;
SELECT 1;
WARNING:  cannot use the hint table
HINT:  Run "CREATE EXTENSION pg_hint_plan" to create the hint table.
 ?column? 
----------
        1
(1 row)

SET pg_hint_plan.enable_hint_table TO off;
CREATE EXTENSION pg_hint_plan;
SET pg_hint_plan.enable_hint_table TO on;
SELECT 1;
 ?column? 
----------
        1
(1 row)

SET pg_hint_plan.enable_hint_table TO off;
DROP EXTENSION pg_hint_plan;
